app/views/unattended/provisioning_templates/snippet/preseed_kernel_options_autoinstall.erb

```diff
@@ -30,7 +35,14 @@ description: options for the kernel / preseed startup initialization
     options << "ip=[#{iface.ip6}]::[#{subnet6.gateway}]:[#{subnet6.mask}]:#{hostname}:#{iface.identifier}:none:[#{subnet6.dns_servers.join(']:[')}]"
   end
 
+  options << 'ramdisk_size=1500000'
+  options << 'fsck.mode=skip'
+  options << 'autoinstall'
+  options << "url=http://#{@preseed_server}#{image_path}"
+  options << 'cloud-config-url=/dev/null'
+  options << "ds=nocloud-net;s=http://#{foreman_request_addr}/userdata/#{mac ? mac + '/' : ''}"
   options << 'console-setup/ask_detect=false'
+  options << "locale=#{host_param('lang') || 'en_US'}"
```

Comment:
This is also a reasonable change, but I would recommend it putting it in a separate PR as well (maybe together with the mac change I commented on before).

Reply:
Happy to do that.